testsuite/integration-arquillian/tests/base/testsuites/database-suite

```diff
@@ -0,0 +1,17 @@
+AccountRestServiceTest
+RequiredActionUpdateProfileTest
+org.keycloak.testsuite.admin.*
```

Comment:
Does this run also subpackages? For example tests like `org.keycloak.testsuite.admin.client.CredentialsTest`? As AFAIK for maven-surefire-plugin it is needed to use something like `org.keycloak.testsuite.admin.**` to run also sub-packages.

One thing is, that there are some classes, which does not test only admin REST API, but also the functionality itself. For example `org.keycloak.testsuite.admin.client.authorization.PolicyEnforcerTest`

Reply:
Updated to '**'.

Didn't count for things like `PolicyEnforcerTest` that's clearly a test that is in the wrong category.

Reply:
Yes, agree. On the other hand, it will be likely good to test CRUD of authorization objects as part of DB tests?

IMO we can move some tests from package `org.keycloak.testsuite.admin.client.authorization` to the `org.keycloak.testsuite.authz` to make sure that:
- Package  `org.keycloak.testsuite.admin.client.authorization` contains tests for CRUD of authorization objects by REST API
- Package `org.keycloak.testsuite.authz` contains tests for authorization itself

Then we can test `org.keycloak.testsuite.admin.client.authorization` package as part of DB tests as well.

Looks like this should be rather done in separate PR?

@pedroigor WDYT?

Reply:
Works for me.

Reply:
Yes, or it can be separate PR, which will be merged prior to this one? Then we will always have DB coverage in the Keycloak main. Whatever you guys prefer.

Reply:
@mposolda +1 to what you are suggesting, that is basically what I figured we should do as well

Reply:
I agree with that as well